seismicpro/batch.py

```diff
@@ -476,6 +476,18 @@ def calculate_metric(self, metric, *args, metric_name=None, coords_component=Non
         self._num_calculated_metrics += 1
         return self
 
+    @staticmethod
+    def _unpack_args(batch_item, args):
+        """Replace all names of batch components in `args` with corresponding batch items. """
```

Comment:
```suggestion
        """Replace all names of batch components in `args` with corresponding values from `batch_item`. """
```
Just to clarify after renaming `batch_item`